package.json

```diff
@@ -74,6 +80,8 @@
   },
   "scripts": {
     "build": "webpack --progress --hide-modules --config  webpack.build.js",
+    "test:unit": "vue-cli-service test:unit",
```

Comment:
👀 could we make do without this since we aren't using `vue-cli`?

Reply:
Yeap, deleted.